### DISCUSSION TITLE
Add an option to disable shortcuts

### DISCUSSION THREAD

**Discussion by mityukov** [alpinejs/alpine#1784] [3. Feature ideas]
Hi, is it possible to add an option allowing to disable shortcuts like `:prop` or `@event`? Such a "strict" mode.

For instance, I still  have plenty of Vue JS components in my project (which I gradually transfer to TALL), and those components are extensively using the same shortcuts. I don't mind using full syntax in all my new code (`x-on:zzz`,  `x-bind:bbb`), but there's lots of the old one which sometimes conflicts…

**Comment by calebporzio** [comment]
Interesting, in general Alpine is not meant to be intermingled with Vue, however, I could see the desire for something like this.

Maybe if there's enough demand for this I would consider adding it, but in the meantime, my recommendation would be to do a find and replace in all your Vue components to use `v-on and v-bind` respectively to keep the tools separate.
